mailchimp3/baseapi.py

```diff
@@ -42,16 +42,28 @@ def _iterate(self, url, **kwargs):
         kwargs['count'] = integer
         kwargs['offset'] = integer
         """
+        #fields as a kwarg ought to be a string with comma-separated substring
+        #values to pass along to self._mc_client._get(). it also ought to
+        #contain 'total_items' in all cases, so that is enforced here
+        if kwargs.get('fields') is None:
+            kwargs['fields'] = 'total_items'
+        else:  # assume well formed string, just missing 'total_items'
+            if not 'total_items' in kwargs['fields'].split(','):
```

Comment:
It looks like it does not work when a list is passed and it must be a comma-delimited string, so ignore this comment. As for the other, there is no complication, we simply check to see if fields is provided, if not we do nothing to change that, and if it is, we make sure that total_items is part of that list of fields. You could probably replace the start of that block with `if kwargs.get('fields') is not None:` and simply don't have an `else` case.